### PR TITLE
workflows: try using linux modules cache in windows e2e test

### PR DESCRIPTION
**Pull request by Rugvip**
Curious if this works :grin:

Could be worth having on the CI e2e test but not master build.

### CODE REVIEW

**Review by freben — APPROVED** [review]
Haha, imaginative - let's try i guess

**Review by soapraj — APPROVED** [review]

File: .github/workflows/e2e-win.yml [hunk 1]
```diff
@@ -33,11 +33,21 @@ jobs:
...
           node-version: ${{ matrix.node-version }}
           registry-url: https://registry.npmjs.org/ # Needed for auth
+      - name: cache all node_modules
+        id: cache-modules
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          # This is intentionally set to Linux, since Windows produces broken node_modules
+          # cache tarballs because it follows symlinks.
+          key: Linux-node_modules-${{ hashFiles('yarn.lock', '**/package.json') }}
```
Comment (shmidt-i) [inline .github/workflows/e2e-win.yml:43]: But why is this in the `e2e-win` file ? 🙃

Comment (freben) [reply .github/workflows/e2e-win.yml:43]: see the comment above :)

Comment (Rugvip) [reply .github/workflows/e2e-win.yml:43]: I tried 😅

**Comment by Rugvip** [comment]
Have a feeling the `hashFiles` also follows symlinks on windows, because the hash ends up being different on Linux and Windows. Trying to switch to a more specific pattern for finding the `package.json`s.

**Comment by Rugvip** [comment]
Meh, realized the hash produced for only `yarn.lock` is different on Windows and Linux. Might be that line endings are checked out differently causing different results. Either way this won't work because of that, so closing.

**Pull request closed without merging**